curriculum/challenges/english/blocks/managing-packages-with-npm/587d7fb3367417b2b2512bfb.md

```diff
@@ -2,7 +2,6 @@
 id: 587d7fb3367417b2b2512bfb
 title: 'How to Use package.json, the Core of Any Node.js Project or npm Package'
 challengeType: 2
-forumTopicId: 301528
```

Comment:
this topic is still available, there is no need to remove this https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-how-to-use-package-json-the-core-of-any-node-js-project-or-npm-package/301528